tests/test_connector.py

```diff
@@ -452,10 +453,30 @@ def test_cleanup3(self):
     def test_tcp_connector_ctor(self):
         conn = aiohttp.TCPConnector(loop=self.loop)
         self.assertTrue(conn.verify_ssl)
+        self.assertIs(conn.verify_fingerprint, None)
         self.assertFalse(conn.resolve)
         self.assertEqual(conn.family, socket.AF_INET)
         self.assertEqual(conn.resolved_hosts, {})
 
+    def test_tcp_connector_verify_fingerprint(self):
+        # sha1 fingerprint of ./sample.crt.der
+        fpgood = '7393fd3aed081d6fa9ae71391ae3c57f89e76cf9'
+        fpbad = 'badbadbadbadbadbadbadbadbadbadbadbadbad1'
+        for fp in (fpgood, fpbad):
+            conn = aiohttp.TCPConnector(loop=self.loop, verify_ssl=False,
+                                        verify_fingerprint=fp)
+            with test_utils.run_server(self.loop, use_ssl=True) as httpd:
+                coro = client.request('get', httpd.url('method', 'get'),
+                                      connector=conn, loop=self.loop)
+                if fp == fpgood:
+                    # should not raise
+                    self.loop.run_until_complete(coro)
+                else:
+                    with self.assertRaises(FingerprintMismatch) as cm:
+                        self.loop.run_until_complete(coro)
+                    self.assertEqual(cm.exception.expected, fpbad)
+                    self.assertEqual(cm.exception.got, fpgood)
+
```

Comment:
Travis tests are looking green. Satisfied with test coverage of the new code? Anything else I can do to get this ready to merge? Thanks again for reviewing.